encord/constants/model.py

```diff
@@ -13,14 +13,76 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-FAST_AI = "fast_ai"
-RESNET18 = "resnet18"
-RESNET34 = "resnet34"
-RESNET50 = "resnet50"
-RESNET101 = "resnet101"
-RESNET152 = "resnet152"
-VGG16 = "vgg16"
-VGG19 = "vgg19"
-YOLOV5 = "yolov5"
-FASTER_RCNN = "faster_rcnn"
-MASK_RCNN = "mask_rcnn"
+from __future__ import annotations
```

Comment:
Are we sure this is not going to break for different python versions?

Reply:
So far, this is what needs to be done for all versions 3.7, 3.8, 3.9, 3.10 - [see here](https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations). 
According to the 3.7 documentation, forward references for typing should have become default in 3.10, but when I check [here](https://docs.python.org/3.11/whatsnew/3.10.html) it doesn't seem to be the case. 
When I look at the python 3.10.4 documentation of  [__future__](https://docs.python.org/3/library/__future__.html) it indicates that `annotations` is mandatory in 3.11, whatever that means. I can't see it implemented in the [3.11](https://docs.python.org/3.11/whatsnew/3.11.html) either. 

Long story short, I think that it is safe to include for now. Although, I have not tested this on all python versions.

Reply:
Thanks for sharing! I used to always do 
```py
class A:
    def func() -> 'A':
```
Which worked for me too, but it wasn't really a forward declaration. It think this looks fine!